layers/+tools/salt/README.org

```diff
@@ -12,7 +12,7 @@ This layer provides syntax highlighting for Saltstack files.
 
```

Comment:
I think it may be best to keep it as saltstack here as the product is known as saltstack to many.